pixart.js

```diff
@@ -1 +1,11 @@
+var form = document.getElementById('form');
+var input = $("#color-field");
+var brush = $(".brush");
+var button = document.getElementById('set-color');
 
+var hello = function() {
+  event.preventDefault();
```

Comment:
`event.preventDefault` should be directly inside the event listener, not the callback function. Otherwise, it won't be called and your button will still have its default behavior.